tools/importPlanGoals.test.js

```diff
@@ -10,37 +11,70 @@ describe('Import TTA plan goals', () => {
   });
   describe('Update grants and grantees', () => {
     beforeAll(async () => {
-      await Grant.destroy({ where: {} });
-      await Grantee.destroy({ where: {} });
+      await Grant.destroy({ where: { id: { [Op.gt]: 20 } } });
```

Comment:
I don't see grants/grantees with ids less than 20 reverenced in these tests, do we need this `greater than` condition?

Reply:
The seeders now have grants and grantees with ids that are small.